modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -65,6 +93,17 @@ INSFVMomentumAdvection::initialSetup()
       std::set<ExecFlagType>({EXEC_ALWAYS}),
       _mesh,
       this->blockIDs());
+
+  if (_absolute_momentum_limiter)
+    _mom_abs = std::make_unique<PiecewiseByBlockLambdaFunctor<ADReal>>(
```

Comment:
alternatively, we use the speedFunctorMaterial to define that one too
I can go either way

Reply:
I think I would prefer that

Reply:
I ll move it if I see anything else use this? maybe when/if FV TH fvkernels get added